modules/solid_mechanics/src/bcs/ExplicitDirichletBCBase.C

```diff
@@ -64,13 +71,33 @@ ExplicitDirichletBCBase::timestepSetup()
   _var_time_order = _explicit_integrator->findVariableTimeOrder(_var.number());
 }
 
+// const NumericVector<Number> &
+// ExplicitDirichletBCBase::initLumpedMass()
+// {
+//   const auto & nl = _fe_problem.getNonlinearSystemBase(_sys.number());
+//   if (nl.hasVector("mass_matrix_diag_inverted"))
+//     return nl.getVector("mass_matrix_diag_inverted");
+
+//   mooseError("Lumped mass matrix is missing. Make sure ExplicitMixedOrder is being used as
+//   the "
+//              "time integrator.");
+// }
+
 const NumericVector<Number> &
-ExplicitDirichletBCBase::initLumpedMass()
+ExplicitDirichletBCBase::initMassLumped()
 {
   const auto & nl = _fe_problem.getNonlinearSystemBase(_sys.number());
-  if (nl.hasVector("mass_matrix_diag_inverted"))
-    return nl.getVector("mass_matrix_diag_inverted");
+  if (nl.hasVector("mass_matrix_lumped"))
+    return nl.getVector("mass_matrix_lumped");
 
   mooseError("Lumped mass matrix is missing. Make sure ExplicitMixedOrder is being used as the "
              "time integrator.");
 }
+
+const NumericVector<Number> &
+ExplicitDirichletBCBase::initDampingLumped()
+{
+  const auto & nl = _fe_problem.getNonlinearSystemBase(_sys.number());
+  // if (nl.hasVector("damping_matrix_lumped"))
+  return nl.getVector("damping_matrix_lumped");
```

Comment:
Same here, if we don't have a damping term, do we need to initialize this vector?